### DISCUSSION TITLE
How to configure RustDesk from command line --option // --config

### DISCUSSION THREAD

**Discussion by jopejuca** [rustdesk/rustdesk#7613] [Q&A] [answered]
Hello, I want to use RustDesk to access linux servers on the cloud. 
I want to be able to configure the servers via command line using either the --config or the --option on the command line args.
But I was unable to find the correct way to enable/disble clipboard or file transfer using those parameters.
Ideally I want to be able to paste and send files into de server but never out of the server. If this is not possible, disbling clipboard and file transfer would be ok. Is there any documentation or specific file in the source code that could help me finding out what are the correct options to pass via command line: `rustdesk --config ??`

**Comment by fufesou** [comment, marked as answer]
`sudo rustdesk --option enable-clipboard N`
https://github.com/rustdesk/rustdesk/blob/bdc5cded221af9697eb29aa30babce75e987fcc9/src/core_main.rs#L314

**Reply by gitterspec** [reply]
The linked documentation doesn't seem to specify values for --option. Looking at the code, --option calls get_options, but going to that function doesn't shed more light. Would appreciate any other tips/insights here.

**Reply by Messere** [reply]
Take a look at https://github.com/rustdesk/rustdesk/blob/1f5aeda41dcd3c1844a3a8328d7221076cbac2db/libs/hbb_common/src/config.rs#L2164 

It's still not as straightforward as a proper documentation could be, but may help you in some cases. For exaple:

```sudo rustdesk --option allow-remote-config-modification Y```

enables remote config modifications on linux.
